packages/main/src/components/FlexBox/FlexBox.jss.ts

```diff
@@ -1,7 +1,11 @@
+import type { CSSProperties } from 'react';
+
 export const styles = {
   flexBox: {
     display: 'flex',
 
+    gap: ({ gap }: { gap: CSSProperties['gap'] }) => (typeof gap === 'number' ? `${gap}rem` : gap),
```

Comment:
**Question:** Shall we consider a different scale for numbers based styles? 

The `gap * 1rem` scaling was the easiest and most straight forward one, that I could come up with at the moment.